src/main/scala/firrtl/stage/FirrtlAnnotations.scala

```diff
@@ -152,22 +152,23 @@ object FirrtlSourceAnnotation extends HasShellOptions {
   *  - If unset, a [[CompilerAnnotation]] with the default [[VerilogCompiler]]
   * @param compiler compiler name
   */
+@deprecated("Use a RunFirrtlTransformAnnotation targeting a specific Emitter.", "FIRRTL 1.4.0")
 case class CompilerAnnotation(compiler: Compiler = new VerilogCompiler()) extends NoTargetAnnotation with FirrtlOption
 
 object CompilerAnnotation extends HasShellOptions {
```

Comment:
Should this object be deprecated and it's contents and moved elsewhere?

Reply:
It's slated for removal in 1.5.0, so yes. I'll do that and move the logic into `RunFirrtlTransformAnnotation$`.

Reply:
Deprecated in later commits.